lib/l10n/intl_es.arb

```
@@ -894,6 +899,60 @@
+  
+  "@_comment_dispute_screen": "Texto de Pantalla de Disputas",
+  "disputeNotFound": "Disputa no encontrada",
+  "errorLoadingDispute": "Error cargando disputa: {error}",
+  "@errorLoadingDispute": {
+    "placeholders": {
+      "error": { "type": "String" }
+    }
+  }
+
+
   "@_comment_relay_status": "Mensajes de estado de relays",
```

RuboCop (1.76.1)

[fatal] unexpected token tSTRING
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)